src/js/stores/ui.js

```diff
@@ -266,6 +266,10 @@ define(function (require, exports, module) {
                 windowWidth = window.document.body.clientWidth,
                 windowHeight = window.document.body.clientHeight;
 
+            if (!centerOffsets) {
```

Comment:
I wonder if this will solve the occasional cloaking weirdness I've seen recently.